src/couchapps/WMStats/_attachments/js/ViewModels/WMStats.ViewModel.js

```diff
@@ -433,6 +433,36 @@ WMStats.ViewModel = (function (){
             summary.SoftwareVersions = requestInfo.SoftwareVersions;
             summary.CMSSWVersion = requestInfo.SoftwareVersions[0];
             summary.Campaign = requestInfo.Campaign;
+            // assign more value for acdc assignment
+            summary.Team = requestInfo.Teams[0];
+            //TODO get the site white list from acdc db. (site might be overflowed)
+            //also change templete to handle multiple selection
+            //summary.SiteWhitelist = requestInfo.SiteWhitelist.join();
+            //summary.SiteBlacklist = requestInfo.SiteBlacklist.join();
+            summary.AcquisitionEra = requestInfo.AcquisitionEra;
+            summary.ProcessingString = requestInfo.ProcessingString;
+            summary.ProcessingVersion = requestInfo.ProcessingVersion;
+            summary.Dashboard = requestInfo.Dashboard;
+            summary.MergedLFNBase = requestInfo.MergedLFNBase;
+            summary.TrustSitelists = false;
```

Comment:
TrustSiteList only applies to the original top-level task. If you rerun jobs from that one, you need to inherit from the request. If you are not (like rerunning some merges), you want to use false.